### ISSUE TITLE
Production flag

### ISSUE THREAD

**Issue by rexxars** [bjoerge/weigh#11]
According to the readme, you can pass `NODE_ENV=production`, to get pass this onto envify et al.

With the latest version of npm, this seems to fail - weigh crashes:

```
espenh@ambrose ~/webdev/weigh $ NODE_ENV=production weigh ascii-doge
⠏ Downloading ascii-doge, this may take a little while…/home/espenh/webdev/weigh/lib/installPackages.js:72
  return Object.keys(dependencies)
                ^

TypeError: Cannot convert undefined or null to object
    at packagesToArray (/home/espenh/webdev/weigh/lib/installPackages.js:72:17)
    at /home/espenh/webdev/weigh/lib/installPackages.js:64:16
    at ChildProcess.exithandler (child_process.js:198:7)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:852:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
```

After some digging, it seems `npm ls` changes behavior when `NODE_ENV` is set to production - but I can't find any documentation stating why, or how. This might be related: https://github.com/npm/npm/issues/12579

Regardless of this, I'm wondering if perhaps it would be a better option to pass a `--production` flag, or similar? In fact, I'd actually prefer if weigh always gave me the size of production bundles, since that is what I'm usually basing my decisions on. Thoughts?

**Comment by bjoerge** [comment]
Thats strange... pretty sure it used to work.

I like the idea of passing `NODE_ENV: production` to envify by default. I can't really think of a situation where I'd want to know the size of the development bundle specifically. The main purpose of `weigh` is to provide an estimate of how many extra bytes a package will add to the bundle, and  the size of the production bundle is probably whats matters to most, so +1 from me.

If it turns out knowing the size of the _development_ bundle is indeed useful in some cases, my guess is that those cases still would be rather rare in comparison, and maybe better be covered by a `--development` flag or similar.

**Comment by bjoerge** [comment]
Implemented a `--env` command line option which is default `production`. Released in v4.0.0 ✨
